### PR TITLE
ticdc: Use open-protocol instead of default protocol and add missing params

### PR DESCRIPTION
**Pull request by hi-rustin**
Signed-off-by: hi-rustin <rustin.liu@gmail.com>

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

* Use open-protocol instead of default protocol.
* Fix some wrong default values.(compressionType/max-batch-size)
* Fix some typos

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v5.3 (TiDB 5.3 versions)
- [x] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [x] Need modification after applied to another branch

### CODE REVIEW

File: ticdc/manage-ticdc.md [hunk 1]
```diff
@@ -208,8 +208,8 @@ URI 中可配置的的参数如下：
 | `partition-num`      | 下游 Kafka partition 数量（可选，不能大于实际 partition 数量，否则创建同步任务会失败，默认值 `3`）|
 | `max-message-bytes`  | 每次向 Kafka broker 发送消息的最大数据量（可选，默认值 `1MB`） |
 | `replication-factor` | kafka 消息保存副本数（可选，默认值 `1`）                       |
-| `protocol` | 输出到 kafka 消息协议，可选值有 `default`、`canal`、`avro`、`maxwell`（默认值为 `default`） |
-| `max-batch-size` |  从 v4.0.9 引入。如果消息协议支持将多条变更记录输出到一条 kafka 消息，该参数指定一条 kafka 消息中变更记录的最多数量，目前仅对 Kafka 的 `protocol` 为 `default` 时有效（可选，默认值为 `16`）|
+| `protocol` | 输出到 kafka 消息协议，可选值有 `open-protocol`、`canal`、`canal-json`、`avro`、`maxwell` |
```
Comment (leoppro) [inline ticdc/manage-ticdc.md:211]:
```suggestion
| `protocol` | 输出到 kafka 消息协议，可选值有 `open-protocol`、`canal`、`canal-json`、`avro`、`maxwell` |
```
把 canal-json 拿到前面来

File: ticdc/manage-ticdc.md [hunk 2]
```diff
@@ -575,8 +575,8 @@ dispatchers = [
     {matcher = ['test3.*', 'test4.*'], dispatcher = "rowid"},
 ]
 # 对于 MQ 类的 Sink，可以指定消息的协议格式
-# 目前支持 default、canal、avro 和 maxwell 四种协议。default 为 TiCDC Open Protocol
-protocol = "default"
+# 目前支持 open-protocol、canal、canal-json、avro 和 maxwell 五种协议。
+protocol = "open-protocol"
```
Comment (leoppro) [inline ticdc/manage-ticdc.md:579]:
```suggestion
protocol = "canal-json"
```
示例配置文件有重要的引导作用

File: ticdc/manage-ticdc.md [hunk 3]
```diff
@@ -575,8 +575,8 @@ dispatchers = [
     {matcher = ['test3.*', 'test4.*'], dispatcher = "rowid"},
 ]
 # 对于 MQ 类的 Sink，可以指定消息的协议格式
-# 目前支持 default、canal、avro 和 maxwell 四种协议。default 为 TiCDC Open Protocol
-protocol = "default"
+# 目前支持 open-protocol、canal、canal-json、avro 和 maxwell 五种协议。
```
Comment (leoppro) [inline ticdc/manage-ticdc.md:578]: canal-json 拿到前面来

File: ticdc/ticdc-open-api.md [hunk 4]
```diff
@@ -144,7 +144,7 @@ curl -X GET http://127.0.0.1:8300/api/v1/health
     {"matcher":["test1.*", "test2.*"], "dispatcher":"ts"},
     {"matcher":["test3.*", "test4.*"], "dispatcher":"rowid"}
   ],
-  "protocal":"default"
+  "protocal":"open-protocol"
```
Comment (leoppro) [inline ticdc/ticdc-open-api.md:147]:
```suggestion
  "protocal":"canal-json"
```

File: ticdc/ticdc-open-api.md [hunk 5]
```diff
@@ -157,7 +157,7 @@ curl -X GET http://127.0.0.1:8300/api/v1/health
 
 `matcher`：匹配语法和过滤器规则语法相同。
 
-`protocal`：对于 MQ 类的 Sink，可以指定消息的协议格式。目前支持 default、canal、avro 和 maxwell 四种协议，默认为 TiCDC Open Protocol。
+`protocol`：对于 MQ 类的 Sink，可以指定消息的协议格式。目前支持 open-protocol、canal、canal-json、avro 和 maxwell 五种协议。
```
Comment (leoppro) [inline ticdc/ticdc-open-api.md:160]: canal-json 拿到前面来

File: ticdc/manage-ticdc.md (hunk 1 above)
Comment (leoppro) [reply ticdc/manage-ticdc.md:211]: 最好在合适的地方提几句，建议使用  canal-json 协议

Comment (3AceShowHand) [reply ticdc/manage-ticdc.md:211]: 能不能直接把 `maxwell` 和 `canal` 给拿掉？

Comment (hi-rustin) [reply ticdc/manage-ticdc.md]: 修复了

File: ticdc/manage-ticdc.md (hunk 3 above)
Comment (hi-rustin) [reply ticdc/manage-ticdc.md]: 修复了

File: ticdc/ticdc-open-api.md (hunk 5 above)
Comment (hi-rustin) [reply ticdc/ticdc-open-api.md]: 修复了

File: ticdc/manage-ticdc.md [hunk 6]
```diff
@@ -221,10 +221,11 @@ URI 中可配置的的参数如下：
 
 * TiCDC 推荐用户自行创建 Kafka Topic，你至少需要设置该 Topic 每次向 Kafka broker 发送消息的最大数据量和下游 Kafka partition 的数量。在创建 changefeed 的时候，这两项设置分别对应 `max-message-bytes` 和 `partition-num` 参数。
 * 如果你在创建 changefeed 时，使用了尚未存在的 Topic，那么 TiCDC 会尝试使用 `partition-num` 和 `replication-factor` 参数自行创建 Topic。建议明确指定这两个参数。
+* 建议使用 canal-json 协议。
```
Comment (leoppro) [inline ticdc/manage-ticdc.md:224]:
```suggestion
* 在大多数情况下，建议使用 canal-json 协议。
```

**Review by leoppro — APPROVED** [review]

**Comment by hi-rustin** [comment]
/hold

Waiting for https://github.com/pingcap/tiflow/pull/4036 landed.

File: ticdc/manage-ticdc.md [hunk 7]
```diff
@@ -206,10 +206,12 @@ URI 中可配置的的参数如下：
 | `kafka-version`      | 下游 Kafka 版本号（可选，默认值 `2.4.0`，目前支持的最低版本为 `0.11.0.2`，最高版本为 `2.7.0`。该值需要与下游 Kafka 的实际版本保持一致） |
 | `kafka-client-id`    | 指定同步任务的 Kafka 客户端的 ID（可选，默认值为 `TiCDC_sarama_producer_同步任务的 ID`） |
 | `partition-num`      | 下游 Kafka partition 数量（可选，不能大于实际 partition 数量，否则创建同步任务会失败，默认值 `3`）|
-| `max-message-bytes`  | 每次向 Kafka broker 发送消息的最大数据量（可选，默认值 `1MB`） |
-| `replication-factor` | kafka 消息保存副本数（可选，默认值 `1`）                       |
-| `protocol` | 输出到 kafka 消息协议，可选值有 `default`、`canal`、`avro`、`maxwell`（默认值为 `default`） |
-| `max-batch-size` |  从 v4.0.9 引入。如果消息协议支持将多条变更记录输出到一条 kafka 消息，该参数指定一条 kafka 消息中变更记录的最多数量，目前仅对 Kafka 的 `protocol` 为 `default` 时有效（可选，默认值为 `16`）|
+| `max-message-bytes`  | 每次向 Kafka broker 发送消息的最大数据量（可选，默认值 `10MB`）。当该值大于 Kafka broker 或者 topic 的 max-message-bytes 时，会取它们中的最小值来控制消息大小。 |
```
Comment (leoppro) [inline ticdc/manage-ticdc.md:209]:
```suggestion
| `max-message-bytes`  | 每次向 Kafka broker 发送消息的最大数据量（可选，默认值 `10MB`）。|
```

**Comment by hi-rustin** [comment]
@TomShawn 
Could you please take a look? Thanks! 💚 💙 💜 💛 ❤️

**Comment by TomShawn** [comment]
> @TomShawn Could you please take a look? Thanks! 💚 💙 💜 💛 ❤️

@hfxsd PTAL thanks!

@hi-rustin Where there is a configuration change or default value change, it is always bound to a specific version (since v..., the default value of xxx changes from .. to ..).

**Comment by hi-rustin** [comment]
> 

Shouldn't these be marked on the release branch?

File: ticdc/manage-ticdc.md [hunk 8]
```diff
@@ -206,10 +206,12 @@ URI 中可配置的的参数如下：
 | `kafka-version`      | 下游 Kafka 版本号（可选，默认值 `2.4.0`，目前支持的最低版本为 `0.11.0.2`，最高版本为 `2.7.0`。该值需要与下游 Kafka 的实际版本保持一致） |
 | `kafka-client-id`    | 指定同步任务的 Kafka 客户端的 ID（可选，默认值为 `TiCDC_sarama_producer_同步任务的 ID`） |
 | `partition-num`      | 下游 Kafka partition 数量（可选，不能大于实际 partition 数量，否则创建同步任务会失败，默认值 `3`）|
-| `max-message-bytes`  | 每次向 Kafka broker 发送消息的最大数据量（可选，默认值 `1MB`） |
-| `replication-factor` | kafka 消息保存副本数（可选，默认值 `1`）                       |
-| `protocol` | 输出到 kafka 消息协议，可选值有 `default`、`canal`、`avro`、`maxwell`（默认值为 `default`） |
-| `max-batch-size` |  从 v4.0.9 引入。如果消息协议支持将多条变更记录输出到一条 kafka 消息，该参数指定一条 kafka 消息中变更记录的最多数量，目前仅对 Kafka 的 `protocol` 为 `default` 时有效（可选，默认值为 `16`）|
+| `max-message-bytes`  | 每次向 Kafka broker 发送消息的最大数据量（可选，默认值 `10MB`）。|
```
Comment (hfxsd) [inline ticdc/manage-ticdc.md:209]: 需要加上此更新从哪些版本开始生效。

File: ticdc/manage-ticdc.md [hunk 9]
```diff
@@ -206,10 +206,12 @@ URI 中可配置的的参数如下：
...
+| `replication-factor` | Kafka 消息保存副本数（可选，默认值 `1`）                       |
+| `protocol` | 输出到 Kafka 消息协议，可选值有 `canal-json`、`open-protocol`、`canal`、`avro`、`maxwell` |
+| `auto-create-topic` | 当传入的 `topic-name` 在 Kafka 集群不存在时，TiCDC 是否自动创建该 topic（可选，默认值为 `true`） |
```
Comment (hfxsd) [inline ticdc/manage-ticdc.md:212]: 需要加上引入的版本。

File: ticdc/manage-ticdc.md [hunk 10]
```diff
@@ -206,10 +206,12 @@ URI 中可配置的的参数如下：
...
+| `enable-tidb-extension` | 输出协议为 `canal-json` 时，如果该值为 true，则发送 Resolved 事件并且在消息中添加 TiDB 扩展字段（可选，默认值 `false`）|
```
Comment (hfxsd) [inline ticdc/manage-ticdc.md:213]: 需要加上引入的版本。

File: ticdc/manage-ticdc.md [hunk 11]
```diff
@@ -271,7 +274,7 @@ URI 中可配置的的参数如下：
 | `maxPendingMessages` | Pending 消息队列的最大大小，例如，等待接收来自 Pulsar 的确认的消息（可选，默认值为 1000） |
 | `disableBatching` | 禁止自动批量发送消息（可选） |
 | `batchingMaxPublishDelay` | 设置发送消息的批处理时间（默认值为 10ms） |
-| `compressionType` | 设置发送消息时使用的压缩算法（可选 `LZ4`，`ZLIB` 和 `ZSTD`，默认值为 `ZSTD`）|
+| `compressionType` | 设置发送消息时使用的压缩算法（可选 `NONE`，`LZ4`，`ZLIB` 和 `ZSTD`，默认值为 `NONE`）|
```
Comment (hfxsd) [inline ticdc/manage-ticdc.md:277]: 需要加上此更新从哪些版本开始生效。

File: ticdc/manage-ticdc.md [hunk 12]
```diff
@@ -206,10 +206,12 @@ URI 中可配置的的参数如下：
 | `kafka-version`      | 下游 Kafka 版本号（可选，默认值 `2.4.0`，目前支持的最低版本为 `0.11.0.2`，最高版本为 `2.7.0`。该值需要与下游 Kafka 的实际版本保持一致） |
 | `kafka-client-id`    | 指定同步任务的 Kafka 客户端的 ID（可选，默认值为 `TiCDC_sarama_producer_同步任务的 ID`） |
 | `partition-num`      | 下游 Kafka partition 数量（可选，不能大于实际 partition 数量，否则创建同步任务会失败，默认值 `3`）|
-| `max-message-bytes`  | 每次向 Kafka broker 发送消息的最大数据量（可选，默认值 `1MB`） |
-| `replication-factor` | kafka 消息保存副本数（可选，默认值 `1`）                       |
-| `protocol` | 输出到 kafka 消息协议，可选值有 `default`、`canal`、`avro`、`maxwell`（默认值为 `default`） |
-| `max-batch-size` |  从 v4.0.9 引入。如果消息协议支持将多条变更记录输出到一条 kafka 消息，该参数指定一条 kafka 消息中变更记录的最多数量，目前仅对 Kafka 的 `protocol` 为 `default` 时有效（可选，默认值为 `16`）|
+| `max-message-bytes`  | 每次向 Kafka broker 发送消息的最大数据量（可选，默认值 `10MB`）。|
+| `replication-factor` | Kafka 消息保存副本数（可选，默认值 `1`）                       |
+| `protocol` | 输出到 Kafka 消息协议，可选值有 `canal-json`、`open-protocol`、`canal`、`avro`、`maxwell` |
```
Comment (hfxsd) [inline ticdc/manage-ticdc.md:211]: 需要加上此更新从哪些版本开始生效。

File: ticdc/manage-ticdc.md (hunk 9 above)
Comment (hi-rustin) [reply ticdc/manage-ticdc.md:212]: 这个是一致都有，但是文档一致漏写了。

File: ticdc/manage-ticdc.md (hunk 11 above)
Comment (hi-rustin) [reply ticdc/manage-ticdc.md:277]: 这个也是一直是这样，这是文档写错了。

File: ticdc/manage-ticdc.md (hunk 12 above)
Comment (hi-rustin) [reply ticdc/manage-ticdc.md:211]: 这个也是一直都有，只是文档漏写了 `canal-json`。

File: ticdc/manage-ticdc.md (hunk 10 above)
Comment (hi-rustin) [reply ticdc/manage-ticdc.md:213]: 这是 v5.0.4 才会发布的功能。

**Review by hi-rustin — COMMENTED** [review]
Self check

File: ticdc/manage-ticdc.md [hunk 13]
```diff
@@ -206,10 +206,12 @@ URI 中可配置的的参数如下：
 | `kafka-version`      | 下游 Kafka 版本号（可选，默认值 `2.4.0`，目前支持的最低版本为 `0.11.0.2`，最高版本为 `2.7.0`。该值需要与下游 Kafka 的实际版本保持一致） |
 | `kafka-client-id`    | 指定同步任务的 Kafka 客户端的 ID（可选，默认值为 `TiCDC_sarama_producer_同步任务的 ID`） |
 | `partition-num`      | 下游 Kafka partition 数量（可选，不能大于实际 partition 数量，否则创建同步任务会失败，默认值 `3`）|
-| `max-message-bytes`  | 每次向 Kafka broker 发送消息的最大数据量（可选，默认值 `1MB`） |
-| `replication-factor` | kafka 消息保存副本数（可选，默认值 `1`）                       |
-| `protocol` | 输出到 kafka 消息协议，可选值有 `default`、`canal`、`avro`、`maxwell`（默认值为 `default`） |
-| `max-batch-size` |  从 v4.0.9 引入。如果消息协议支持将多条变更记录输出到一条 kafka 消息，该参数指定一条 kafka 消息中变更记录的最多数量，目前仅对 Kafka 的 `protocol` 为 `default` 时有效（可选，默认值为 `16`）|
+| `max-message-bytes`  | 每次向 Kafka broker 发送消息的最大数据量（可选，默认值 `10MB`）。从 v5.0.6 和 v4.0.6 开始默认值从 64MB 调整至 10MB。|
```
Comment (en-jin19) [inline ticdc/manage-ticdc.md:209]:
```suggestion
| `max-message-bytes`  | 每次向 Kafka broker 发送消息的最大数据量（可选，默认值 `10MB`）。从 v5.0.6 和 v4.0.6 开始，默认值从 64MB 调整至 10MB。|
```

File: ticdc/manage-ticdc.md [hunk 14]
```diff
@@ -206,10 +206,12 @@ URI 中可配置的的参数如下：
...
+| `replication-factor` | Kafka 消息保存副本数（可选，默认值 `1`）                       |
```
Comment (en-jin19) [inline ticdc/manage-ticdc.md:210]:
```suggestion
| `replication-factor` | 保持 Kafka 消息的副本数（可选，默认值 `1`）                       |
```

File: ticdc/manage-ticdc.md [hunk 15]
```diff
@@ -206,10 +206,12 @@ URI 中可配置的的参数如下：
...
+| `protocol` | 输出到 Kafka 消息协议，可选值有 `canal-json`、`open-protocol`、`canal`、`avro`、`maxwell` |
```
Comment (en-jin19) [inline ticdc/manage-ticdc.md:211]:
```suggestion
| `protocol` | 输出到 Kafka 消息的协议，可选值有 `canal-json`、`open-protocol`、`canal`、`avro`、`maxwell` |
```

File: ticdc/manage-ticdc.md [hunk 16]
```diff
@@ -206,10 +206,12 @@ URI 中可配置的的参数如下：
...
+| `auto-create-topic` | 当传入的 `topic-name` 在 Kafka 集群不存在时，TiCDC 是否自动创建该 topic（可选，默认值为 `true`） |
```
Comment (en-jin19) [inline ticdc/manage-ticdc.md:212]:
```suggestion
| `auto-create-topic` | 当传入的 `topic-name` 不在 Kafka 集群中时，TiCDC 是否自动创建该 topic（可选，默认值为 `true`） |
```

File: ticdc/manage-ticdc.md [hunk 17]
```diff
@@ -206,10 +206,12 @@ URI 中可配置的的参数如下：
...
+| `enable-tidb-extension` | 输出协议为 `canal-json` 时，如果该值为 true，则发送 Resolved 事件并且在消息中添加 TiDB 扩展字段（可选，默认值 `false`）|
```
Comment (en-jin19) [inline ticdc/manage-ticdc.md:213]: 请问这里的 ”Resolved 事件“ 具体指什么呢？

```suggestion
| `enable-tidb-extension` | 当输出协议为 `canal-json` 时，如果该值为 `true`，TiCDC 会发送 Resolved 事件，并在 Kafka 消息中添加 TiDB 扩展字段（可选，默认值 `false`）|
```

File: ticdc/manage-ticdc.md [hunk 18]
```diff
@@ -206,10 +206,12 @@ URI 中可配置的的参数如下：
...
+| `max-batch-size` |  从 v4.0.9 引入。如果消息协议支持将多条变更记录输出到一条 Kafka 消息，该参数指定一条 Kafka 消息中变更记录的最多数量，目前仅对 Kafka 的 `protocol` 为 `open-protocol` 时有效（可选，默认值为 `16`）|
```
Comment (en-jin19) [inline ticdc/manage-ticdc.md:214]:
```suggestion
| `max-batch-size` |  从 v4.0.9 开始引入。当 Kafka 消息协议支持把多条变更记录输出至一条 Kafka 消息时，该参数用于指定这一条 Kafka 消息中变更记录的最多数量。目前，仅当 Kafka 的 `protocol` 为 `open-protocol` 时有效（可选，默认值 `16`）|
```

File: ticdc/manage-ticdc.md [hunk 19]
```diff
@@ -221,10 +223,11 @@ URI 中可配置的的参数如下：
 
 * TiCDC 推荐用户自行创建 Kafka Topic，你至少需要设置该 Topic 每次向 Kafka broker 发送消息的最大数据量和下游 Kafka partition 的数量。在创建 changefeed 的时候，这两项设置分别对应 `max-message-bytes` 和 `partition-num` 参数。
 * 如果你在创建 changefeed 时，使用了尚未存在的 Topic，那么 TiCDC 会尝试使用 `partition-num` 和 `replication-factor` 参数自行创建 Topic。建议明确指定这两个参数。
+* 在大多数情况下，建议使用 canal-json 协议。
```
Comment (en-jin19) [inline ticdc/manage-ticdc.md:226]:
```suggestion
* 在大多数情况下，建议使用 `canal-json` 协议。
```

File: ticdc/ticdc-open-api.md [hunk 20]
```diff
@@ -157,7 +157,7 @@ curl -X GET http://127.0.0.1:8300/api/v1/health
 
 `matcher`：匹配语法和过滤器规则语法相同。
 
-`protocal`：对于 MQ 类的 Sink，可以指定消息的协议格式。目前支持 default、canal、avro 和 maxwell 四种协议，默认为 TiCDC Open Protocol。
+`protocol`：对于 MQ 类的 Sink，可以指定消息的协议格式。目前支持 canal-json、open-protocol、canal、avro 和 maxwell 五种协议。
```
Comment (en-jin19) [inline ticdc/ticdc-open-api.md:160]:
```suggestion
`protocol`：对于 MQ 类的 Sink，可以指定消息的协议格式。目前支持 `canal-json`、`open-protocol`、`canal`、`avro` 和 `maxwell` 五种协议。
```

File: ticdc/troubleshoot-ticdc.md [hunk 21]
```diff
@@ -273,7 +273,7 @@ cdc cli changefeed create --pd=http://10.0.10.25:2379 --sink-uri="kafka://127.0.
 
 ## TiCDC 把数据同步到 Kafka 时，能在 TiDB 中控制单条消息大小的上限吗？
 
-可以通过 `max-message-bytes` 控制每次向 Kafka broker 发送消息的最大数据量（可选，默认值 64MB）；通过 `max-batch-size` 参数指定每条 kafka 消息中变更记录的最大数量，目前仅对 Kafka 的 protocol 为 `default` 时有效（可选，默认值为 `4096`）。
+可以通过 `max-message-bytes` 控制每次向 Kafka broker 发送消息的最大数据量（可选，默认值 10MB）；通过 `max-batch-size` 参数指定每条 kafka 消息中变更记录的最大数量，目前仅对 Kafka 的 protocol 为 `open-protocol` 时有效（可选，默认值为 `16`）。
```
Comment (en-jin19) [inline ticdc/troubleshoot-ticdc.md:276]:
```suggestion
可以通过 `max-message-bytes` 控制每次向 Kafka broker 发送消息的最大数据量（可选，默认值 10MB）；通过 `max-batch-size` 参数指定每条 kafka 消息中变更记录的最大数量，目前仅对 Kafka 的 `protocol` 为 `open-protocol` 时有效（可选，默认值 `16`）。
```

File: ticdc/troubleshoot-ticdc.md [hunk 22]
```diff
@@ -351,7 +351,7 @@ TiCDC 对大事务（大小超过 5 GB）提供部分支持，根据场景不同
 
 ## TiCDC 集群升级到 v4.0.8 之后，changefeed 报错 `[CDC:ErrKafkaInvalidConfig]Canal requires old value to be enabled`，为什么？
 
-自 v4.0.8 起，如果 changefeed 使用 canal 或者 maxwell 协议输出，TiCDC 会自动开启 Old Value 功能。但如果 TiCDC 是从较旧版本升级到 v4.0.8 或以上版本的，changefeed 使用 canal 或 maxwell 协议的同时 Old Value 功能被禁用，此时会出现该报错。可以按照以下步骤解决该报错：
+自 v4.0.8 起，如果 changefeed 使用 canal-json、canal 或者 maxwell 协议输出，TiCDC 会自动开启 Old Value 功能。但如果 TiCDC 是从较旧版本升级到 v4.0.8 或以上版本的，changefeed 使用 canal-json、canal 或 maxwell 协议的同时 Old Value 功能被禁用，此时会出现该报错。可以按照以下步骤解决该报错：
```
Comment (en-jin19) [inline ticdc/troubleshoot-ticdc.md:354]:
```suggestion
自 v4.0.8 起，如果 changefeed 使用 `canal-json`、`canal` 或者 `maxwell` 协议输出，TiCDC 会自动开启 Old Value 功能。但是，当 TiCDC 是从较旧版本升级到 v4.0.8 或以上版本时，在 changefeed 使用 `canal-json`、`canal` 或 `maxwell` 协议的同时 TiCDC 的 Old Value 功能被禁用。此时，会出现该报错。可以按照以下步骤解决该报错：
```

File: ticdc/manage-ticdc.md (hunk 14 above)
Comment (hi-rustin) [reply ticdc/manage-ticdc.md:210]: 保持用在不太合适，这里说的是数据存储，所以保存可能更切合。

File: ticdc/manage-ticdc.md (hunk 16 above)
Comment (hi-rustin) [reply ticdc/manage-ticdc.md:212]: `不在`可能用在这个地方不太合适，用存在更合适一些。因为这是个虚拟的概念，用在不在可能挺奇怪的。

File: ticdc/manage-ticdc.md (hunk 17 above)
Comment (hi-rustin) [reply ticdc/manage-ticdc.md:213]: v5.4.0 引入，还没发布。

**Comment by hi-rustin** [comment]
@en-jin19 
Thanks for your review!  💚 💙 💜 💛 ❤️

**Review by leoppro — APPROVED** [review]

**Comment by leoppro** [comment]
LGTM

**Review by en-jin19 — APPROVED** [review]
LGTM

**Review by hfxsd — APPROVED** [review]
LGTM

**Pull request merged**